conf/messages

```diff
@@ -84,12 +84,14 @@ error.year.length = Year must be 4 numbers
 
 entryDetails.pageTitle = Entry details
 entryDetails.title = Entry details
-entryDetails.epu.label = EPU number
-entryDetails.epu.hint = This is 3 numbers, for example ‘121’
+entryDetails.paragraph = These can be found on the E2.
```

Comment:
I think this sounds better as entryDetails.representativeParagraph and entryDetails.importerParagraph